tests/test_onnx/utils.py

```diff
@@ -48,9 +53,21 @@ def ort_validate(model, feats, onnx_io='tmp.onnx'):
             verbose=False,
             opset_version=11)
 
-    onnx_outputs = verify_model(feats)
+    if isinstance(feats, tuple):
+        ort_feats = []
+        for feat in feats:
+            ort_feats += feat
+    else:
+        ort_feats = feats
+    onnx_outputs = verify_model(ort_feats)
```

Comment:
`verify_model` function basically only gets the onnx outputs, which means the name `verify_model` is misleading. Please change to a more self-explaining name for this function.